cmd/genji/dbutil/dump.go

```diff
@@ -53,7 +54,7 @@ func Dump(ctx context.Context, db *genji.DB, w io.Writer, tables ...string) erro
 			return err
 		}
 
-		return dumpTable(tx, tableName, w)
+		return dumpTable(tx, bufio.NewWriter(w), tableName)
```

Comment:
I don't think there is a point into:
- passing around the bufio.Writer as param (l.69)
- using the bufio writer after we already written to it (l19 to 56)

We should:

- create a bufio.Writer l19 and l104
- pass io.Writer everywhere
- use `fmt.Fprint*` everywhere